FlightPlanProject/UI/BurnTimeOption.cs

```diff
@@ -0,0 +1,325 @@
+using KSP.Game;
+using KSP.Sim.impl;
+using MuMech;
+using UnityEngine;
+using FlightPlan.KTools.UI;
+
+namespace FlightPlan;
+
+internal class BurnTimeOption
```

Comment:
Choose of burn time is quite complex in fact. but all gathered here

there could be some bug here. a good test could be usefull here